### ISSUE TITLE
Failed to load schema [pls help]

### ISSUE THREAD

**Issue by nirmaanlabs** [dotansimha/graphql-code-generator#8136]
### Describe the bug

I am unable to create typescript types from graphql schema. 
Discussed here [discuss](https://github.com/dotansimha/graphql-code-generator/discussions/8127)

Also, I found out that, it works fine on [stackbliztt](https://stackblitz.com/edit/github-hak4gd-kbetbk?file=codegen.yml,package.json) if i run below command
`npm run codegen`

### Your Example Website or App

https://github.com/nirmaanlabs/bug

### Steps to Reproduce the Bug or Issue

`npm i`
`npm run codegen`

### Expected behavior

I should get a file with generated types

### Screenshots or Videos

[video codegen failed to read schema.zip](https://github.com/dotansimha/graphql-code-generator/files/9173939/video.codegen.failed.to.read.schema.zip)

### Platform

- OS: [Windows]
- NodeJS: [16.16.0]
- `graphql` version: [16.5.0]

- "@graphql-codegen/cli": "^2.9.1",
    "@graphql-codegen/typed-document-node": "^2.3.2",
    "@graphql-codegen/typescript": "^2.7.2",
    "@graphql-codegen/typescript-operations": "^2.5.2",

### Codegen Config File

schema: schema.graphql
generates:
  ./src/generated.ts:
    plugins:
      - typescript
      - typescript-operations
      - typed-document-node

### Additional context

Coming here from the [discussion](https://github.com/dotansimha/graphql-code-generator/discussions/8127)

**Comment by ardatan** [comment]
We need to have a clear reproduction that shows the issue. Otherwise we cannot help you.

**Comment by nirmaanlabs** [comment]
> We need to have a clear reproduction that shows the issue on that StackBlitz example. Otherwise, we cannot help you, sorry.

Hey, @ardatan I have provided the video and GitHub link to my repo, is that not sufficient? 

Is there any way, I can see some more insightful logs of why my schema load command failed? As of now it just tells you failed to load schema, and it does not tell you why did it fail

**Comment by GraemeF** [comment]
I agree, it would be helpful if there was some more information about the error.

**Comment by kav** [comment]
Saw similar and had to do much the same, the verbose and debug flags don't emit the error. Is there a bug in the output code?

**Comment by Parables** [comment]
if your GraphQL server is up and running, you can use the following to print the schema to a file:

```
npm install -g graphqurl
 
gq < graphql-endpoint > --introspect > schema.graphql
 
# if your graphql endpoint requires authentication:
gq < graphql-endpoint > -H "Authorization: Bearer <token>" --introspect > schema.graphql
```

Let me know if it helps you out
